src/Snt/Capi/Repository/Article/FindParameters.php

```diff
@@ -118,4 +169,23 @@ public function buildArticleIdsString($separator = self::DEFAULT_SEPARATOR)
 
         return '';
     }
+
+    /**
+     * @return string
+     */
+    public function buildQuery()
+    {
+        $query = [];
+
+        if (!is_null($this->limit)) {
+            $query[] = sprintf('limit=%s', $this->limit);
+        }
+
+        if ($this->timeRange instanceof TimeRangeParameter) {
+            $query[] = sprintf('since=%s', $this->timeRange->getSince()->format(self::QUERY_DATETIME_FORMAT));
+            $query[] = sprintf('until=%s', $this->timeRange->getUntil()->format(self::QUERY_DATETIME_FORMAT));
+        }
+
+        return implode('&', $query);
```

Comment:
can be done even better using http://php.net/manual/en/function.http-build-query.php

:dancer: :dancer: